spyder/plugins/ipythonconsole.py

```diff
@@ -460,9 +460,9 @@ def setup_page(self):
         run_lines_group = QGroupBox(_("Run code"))
         run_lines_label = QLabel(_("You can run several lines of code when "
                                    "a console is started. Please introduce "
-                                   "each one separated by commas, for "
-                                   "example:<br>"
-                                   "<i>import os, import sys</i>"))
+                                   "each one separated by semicolons and a "

```

Comment:
The semicolon and a space are only needed for python 2.7. Python 3 works without the space

Reply:
Ok, I'll add that to our release noes for beta1 when I merge this one.